### DISCUSSION TITLE
Installation on go1.19.4

### DISCUSSION THREAD

**Discussion by karlfunk-ydb** [jackc/pgx#1462] [General]
Required Caveat: am a GO novice, this is my first GO programming....

Have just installed go1.19.4 on my mac

`go version
go version go1.19.4 darwin/amd64
`

Following the instructions in this projects wiki I try to install using `go get github.com/jackc/pgx/v5`

Which results in: 

`go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.`
	
Trying `go install github.com/jackc/pgx/v5@latest`

Results in: 

`go: downloading github.com/jackc/pgx/v5 v5.2.0
go: downloading github.com/jackc/pgx v3.6.2+incompatible
go: downloading github.com/jackc/pgpassfile v1.0.0
go: downloading golang.org/x/crypto v0.0.0-20220829220503-c86fa9a7ed90
go: downloading github.com/jackc/pgservicefile v0.0.0-20200714003250-2b9c44734f2b
go: downloading golang.org/x/text v0.3.8
package github.com/jackc/pgx/v5 is not a main package`

What is my easiest path forward?

**Comment by jackc** [comment]
> go: go.mod file not found in current directory or any parent directory.

Did you run `go mod init` in your current directory?

**Reply by karlfunk-ydb** [reply]
That works!
Sorry, I get the feeling this was a really dumb Q...

Thanks.
